examples/tf_job.yaml

```diff
@@ -6,6 +6,7 @@ spec:
   replicaSpecs:
     - replicas: 1
       tfReplicaType: MASTER
+      TfPort: 2222
```

Comment:
This seems like a regression in the refactor we should be providing a default port  can you fix that?

Reply:
SGTM, I am closing the PR